deployments/examples/owncloud10_with_oc_web/config/ocis/identifier-registration.dist.yaml

```
@@ -34,7 +34,9 @@ clients:
   - id: xdXOt13JKxym1B1QcEncf2XDkLAexMBFwiT9j6EfhhHFJhs2KM9jbjTmf8JBXE69
     secret: UBntmLjC2yYCeHwsyj73Uwo9TAaecAetRwMw0xYcvNL9yRdLSUi0hUAHfvCHFeFh
     application_type: native
-    insecure: true
+    redirect_uris:
+      - http://127.0.0.1
+      - http://localhost
 
   - id: e4rAsNUSIUs0lF4nbv9FmCeUkTlV9GdgTLDH1b5uie7syb90SzEVrbN7HIpmWJeD
     secret: dInFYGV33xKzhbRmpqQltYNdfLdJIfJ9L5ISoKhNoT9qZftpdWSP71VrpGR9pmoD
```

```suggestion
    secret: dInFYGV33xKzhbRmpqQltYNdfLdJIfJ9L5ISoKhNoT9qZftpdWSP71VrpGR9pmoD
    name: ownCloud Android app
```

Please also add the name to the iOS section.